src/Model/Filter/Boolean.php

```diff
@@ -0,0 +1,40 @@
+<?php
+namespace Search\Model\Filter;
+
+class Boolean extends Base
+{
+
+    /**
+     * Default configuration.
+     *
+     * @var array
+     */
+    protected $_defaultConfig = [
+        'truthy' => [1, true, '1', 'true',  'yes', 'on'],
```

Comment:
I would make those class const

    const TRUTHY = 'truthy';
    const FALSY = 'falsy';

Then one can use $this->config(static::TRUTHY) etc